js/annotations/eventEmitterMixin.js

```diff
@@ -98,23 +110,44 @@ var eventEmitterMixin = {
         prevChartY = e.chartY;
         emitter.cancelClick = false;
         emitter.chart.hasDraggedAnnotation = true;
-        emitter.removeDrag = addEvent(H.doc, 'mousemove', function (e) {
-            emitter.hasDragged = true;
-            e = pointer.normalize(e);
-            e.prevChartX = prevChartX;
-            e.prevChartY = prevChartY;
-            fireEvent(emitter, 'drag', e);
-            prevChartX = e.chartX;
-            prevChartY = e.chartY;
-        });
-        emitter.removeMouseUp = addEvent(H.doc, 'mouseup', function (e) {
-            emitter.cancelClick = emitter.hasDragged;
-            emitter.hasDragged = false;
-            emitter.chart.hasDraggedAnnotation = false;
-            // ControlPoints vs Annotation:
-            fireEvent(pick(emitter.target, emitter), 'afterUpdate');
-            emitter.onMouseUp(e);
-        });
+        if (Highcharts.isTouchDevice) {
+            emitter.removeDrag = addEvent(H.doc, 'touchmove', function (e) {
```

Comment:
Instead of duplicate the block of code, can we use: 

`const dragType = Highcharts.isTouchDevice ? 'touchmove' : 'mousemove'`

and then use it as a param od addEvent?

Reply:
All is corrected and working - thanks for this info!